types/src/zkid.rs

```diff
@@ -32,12 +36,12 @@ pub const IDC_NUM_BYTES: usize = 32;
 // TODO(ZkIdGroth16Zkp): add some static asserts here that these don't exceed the MAX poseidon input sizes
 // TODO(ZkIdGroth16Zkp): determine what our circuit will accept
 
-pub const MAX_EPK_BYTES: usize = 93; // Supports public key lengths of up to 93 bytes.
-pub const MAX_ISS_BYTES: usize = 248;
-pub const MAX_AUD_VAL_BYTES: usize = 248;
-pub const MAX_UID_KEY_BYTES: usize = 248;
-pub const MAX_UID_VAL_BYTES: usize = 248;
-pub const MAX_JWT_HEADER_BYTES: usize = 248;
+pub const MAX_EPK_BYTES: usize = 3 * poseidon_bn254::BYTES_PACKED_PER_SCALAR; // Supports public key lengths of up to 93 bytes.
+pub const MAX_ISS_BYTES: usize = 5 * poseidon_bn254::BYTES_PACKED_PER_SCALAR;
```

Comment:
Why is this specified as 5 * BYTES_PACKED_PER_SCALAR rather than directly?  For `MAX_EPK_BYTES` I understand, but for this one, not sure.

Reply:
the reason is we'd ultimate never choose a value that is not a multiple of BYTES_PACKED_PER_SCALAR right?